src/Umbraco.Core/ContentApps/UserEditorContentAppFactory.cs

```diff
@@ -0,0 +1,33 @@
+using System.Collections.Generic;
+using Umbraco.Cms.Core.Models.ContentEditing;
+using Umbraco.Cms.Core.Models.Membership;
+using Umbraco.Cms.Core.Services;
+using Umbraco.Extensions;
+
+namespace Umbraco.Cms.Core.ContentApps
+{
+    public class UserEditorContentAppFactory : IContentAppFactory
+    {
+        internal const int Weight = -100;
+
+        private ContentApp? _app;
+
+        public ContentApp? GetContentAppFor(object source, IEnumerable<IReadOnlyUserGroup> userGroups)
+        {
+            switch (source)
+            {
+                case IUser _:
+                    return _app ??= new ContentApp()
+                    {
+                        Alias = "umbUser",
+                        Name = "Detail",
```

Comment:
Can we localize this?

Reply:
Updated. The Aliasses are automatically used in translations, so I added the information to the translation file